### DISCUSSION TITLE
how do I customize oh-my-zsh prompt?

### DISCUSSION THREAD

**Discussion by debmowry** [BretFisher/ama#200] [Q&A]
How do you change the prompt in iterm after you've installed oh-my-zsh? Surely it shouldn't take pages and pages of code to figure this out. I had it the way I wanted via .bash-profile but oh-my-zsh obviously doesn't use it.

**Comment by BretFisher** [comment]
First learn a bit on oh-my-zsh customization: https://github.com/robbyrussell/oh-my-zsh/wiki/Customization

Then learn about built-in themes and adding external themes.
https://github.com/robbyrussell/oh-my-zsh/wiki/Themes
https://github.com/robbyrussell/oh-my-zsh/wiki/External-themes

Then if you liked mine, which is designed for presentation, you can see that listed in https://www.bretfisher.com/shell

**Comment by isaacdeleon** [comment]
If i want your theme a need to configure to my self or how can install it ? regards

**Comment by BretFisher** [comment]
Hi @isaacdeleon I'm not sure I understand the question. My custom theme is available on my /shell link, which is oh-my-zsh specific, and their documentation links above will give you the info on how to install custom themes.

**Comment by isaacdeleon** [comment]
Sorry , my question is how can install your theme , like it alot but in your page(https://www.bretfisher.com/shell/) i dont see where are the instructions to install it

by the way i already have ohh my sh with iterm2

**Comment by BretFisher** [comment]
https://github.com/robbyrussell/oh-my-zsh/wiki/Customization#overriding-and-adding-themes

**Comment by isaacdeleon** [comment]
hi Bret where can i write you a question about docker ? thanks regards bro ?

**Comment by BretFisher** [comment]
Create a new issue in this repo
